cmake/Findnl-3.cmake

```diff
@@ -0,0 +1,23 @@
+find_package(PkgConfig REQUIRED)
+
+find_library(NL3_LIBRARY "nl-3")
+if(NOT NL3_LIBRARY)
+	message(FATAL_ERROR "libnl-3 not found!")
+endif()
+
+pkg_check_modules(LIBNL3 libnl-3.0 REQUIRED)
+
```

Comment:
You should use find_package_handle_standard_args, like this:
```cmake
include(FindPackageHandleStandardArgs)
FIND_PACKAGE_HANDLE_STANDARD_ARGS(nl-3 DEFAULT_MSG 
    LIBNL3_LIBRARIES LIBNL3_INCLUDE_DIRS)
```

Reply:
Tried this but it didn't work for me in Ubuntu, we can and should standardize our find module files but I feel this should be done in a separate task.

Reply:
So the changes to FindReadline.cmake that I made in the controller also don't work on Ubuntu then? It should work starting from CMake 3.0 (at least, there is no older version history), according to the documentation...

Reply:
They do - I just tried your branch. Probably I did something wrong locally..